po/it.po

```diff
@@ -511,10 +511,8 @@ msgid "salut! French translations thanks to Nathan (@NathanBnm)"
 msgstr "salut! Traduzioni Francesi grazie a Nathan (@NathanBnm)"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:217
-msgid ""
-"Olá! Brazilian Portuguese translations thanks to Lucas Sanchez dos Anjos"
-msgstr ""
-"Olá! Traduzioni in Portoghese Brasiliano grazie a Lucas Sanchez dos Anjos"
+msgid "Olá! Brazilian Portuguese translations thanks to Lucas Sanchez dos Anjos"
+msgstr "Olá! Traduzioni in Portoghese Brasiliano grazie a Lucas Sanchez dos Anjos"
```

Comment:
Language names should be written in lowercase in Italian.

Reply:
True, I made I've coosed to keep the upper-case letter as a way to highlight the word in the sentence. Fixing this asap.